xml/System.Windows.Forms/ListViewItemStateImageIndexConverter.xml

```diff
@@ -76,8 +76,9 @@
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets a value that indicates whether a <see langword="none" /> or <see langword="null" /> value is valid in the <see cref="T:System.ComponentModel.TypeConverter.StandardValuesCollection" /> collection.</summary>
+        <value>
+          Always returns <see langword="false" /> to indicate that a <see langword="none" /> or <see langword="null" /> value isn't valid in the standard values collection.</value>
```

Comment:
Should it be `...that A or B *values aren't* valid...`?

Reply:
It could be. But @rpetrusha had a good question about what is a none value in this case?

Reply:
`none` is not a key word, please remove the markup.

Reply:
```suggestion
          Always returns <see langword="false" /> to indicate that a <see langword="null" /> value isn't valid in the standard values collection.</value>
```